content/applications/websites/livechat/reports.rst

```diff
@@ -0,0 +1,176 @@
+  either a negative or neutral rating, they are considered *unhappy*.
+- :guilabel:`Count`: the total number of sessions.
+
+.. _livechat/operator-analysis:
+
+Operator Analysis
+-----------------
+
+The *Operator Analysis* report is used to monitor the performance of individual live chat operators.
+The default view for this report is a *bar chart*, which displays only conversations from the
+current month. Conversations are grouped by operator.
+
+To view a different measure, click the :guilabel:`Measures` drop-down at the top-left of the report.
+The measures available for this report include:
+
+- :guilabel:`# of Sessions`: the number of sessions an operator participated in. This measure is
+  included by default.
+- :guilabel:`Average duration`: the average duration of a conversation, in seconds.
+- :guilabel:`Average rating`: the average rating received by the operator.
+- :guilabel:`Time to answer`: the average amount of time before the operator responds to a chat
+  request, in seconds.
+- :guilabel:`Count`: the total number of sessions.
+
+.. image:: reports/operator-analysis.png
+   :align: center
+   :alt: Example of the Operator Analysis report from the Live Chat application.
+
+View and filter options
+=======================
+
+On any Odoo report, the view and filter options vary, depending on what data is being analyzed,
+measured, and grouped. See below for additional information on the available views for the *Live
+Chat* reports.
+
+.. note::
+   The :ref:`Sessions History <livechat/sessions-history>` report is **only** available in *list
+   view*.
+
+Pivot view
+----------
+
+The *pivot* view presents data in an interactive manner. The :ref:`Session Statistics
+<livechat/session-statistics>` and :ref:`Operator Analysis <livechat/operator-analysis>` reports are
+available in pivot view.
+
+The pivot view can be accessed on a report by selecting the :guilabel:`grid icon` at the top-right
+of the screen.
+
+To add a group to a row or column to the pivot view, click the :guilabel:`➕ (plus sign)` next to
+:guilabel:`Total`, and then select one of the groups. To remove one, click the :guilabel:`➖ (minus
+sign)` and de-select the appropriate option.
+
+Graph view
+----------
+
+The *graph* view presents data in either a *bar*, *line*, or *pie* chart. The :ref:`Session
+Statistics <livechat/session-statistics>` and :ref:`Operator Analysis <livechat/operator-analysis>`
+reports are available in graph view.
+
+Switch to the graph view by selecting the :guilabel:`line chart icon` at the top-right of the
+screen. To switch between the different charts, select the *related icon* at the top-left of the
+chart, while in graph view.
+
+.. tip::
+   Both the bar chart and line chart can utilize the stacked view option. This presents two or more
+   groups of data on top of each other, instead of next to each other, making it easier to compare
+   data.
```

Comment:
```suggestion
   Both the bar chart and line chart can utilize the *stacked* view option. This presents two or more
   groups of data on top of each other, instead of next to each other, making it easier to compare
   data.
```